### PR TITLE
Re-add dropped encoding section

### PR DESCRIPTION
**Pull request by garemoko**
See https://github.com/adlnet/xAPI-Spec/blob/1.0.3/xAPI.md#encoding

I have a feeling we said on the call that we'd create a strings data type section and put this there, but I don't think that's right. I think this is broader than strings as values of properties and includes strings containing JSON objects.

### CODE REVIEW

**Comment by andyjohnson** [comment]
Great find!  +1 (leaving open just in case I missed a reason that we would have taken it out...can't imagine what it would be)

File: xAPI.md [hunk 1]
```diff
@@ -4539,6 +4540,13 @@ property MUST occur only once.
 * An LRS MUST NOT reject requests based on their version header as would otherwise be 
 required by <a href="#versioning"/>Versioning</a>.
 
+<a name="encoding"/> 
+
+### 2.3 Encoding
+
+###### Requirement
+* All strings MUST be encoded and interpreted as UTF-8.
```
Comment (fugu13) [inline xAPI.md]: I know we're just re-adding, but some ideas on making this more precise:

* All JSON MUST be encoded and interpreted as UTF-8.
* All strings described in the specification for use outside of JSON documents, such as request parameters, MUST be encoded and interpreted as UTF-8

The reason for the split is, from a technical perspective, JSON encoding doesn't apply to the individual strings within JSON, but to the JSON representation as a whole

Comment (garemoko) [inline xAPI.md]: I'm not going to do this one as part of this PR, but if you want to do it as a separate PR (perhaps branched off garemoko:issue-798-1-4 or just done from scratch) I would +1 it.

**Comment by andyjohnson** [comment]
It sounds like @fugu13 agrees with what is here, but can add preciseness.  Merging as it has been through 2 sets of eyes and to allow a potential precision-patch to happen without conflict.
